core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -89,7 +89,7 @@ class ConsoleConsumerTest extends JUnitSuite {
     val config = new ConsoleConsumer.ConsumerConfig(args)
 
     //Then
-    assertFalse(config.useNewConsumer)
+    assertTrue(config.useOldConsumer)
```

Comment:
The `useNewConsumer` boolean flag is removed.  `useOldConsumer` flag is used instead so the assertion is just inverted.